taskcluster/ci/test/kind.yml

```diff
@@ -42,3 +42,24 @@ jobs:
         run-on-tasks-for: [github-push, github-pull-request]
         run:
             gradlew: ['clean', 'testDebugUnitTest']
+    ui:
```

Comment:
If you move this to a new kind, you wouldn't need to adjust the `rb_taskgraph.build` transform; I don't think it is doing anything useful here.

Having said that, I think the only useful thing it is doing for any of the stanzas, is the `include-nightly-version`. It might be worthwhile to separate that out (or call it directly; I'm not sure if that would be confusing, though), and just drop the build transform here. I suspect that the use here may date from when things were split by ABI, and its current use is vestigal.

Reply:
I was on the fence about it too. Then, I remember all gecko tests are under `taskcluster/ci/test`. For the sake of parity, I think it's good to keep them there for now. I'm happy to revisit later. What do you think?

Reply:
If we keep them in the same kind, I think it would make sense to stop using the `rb_taskgraph.build` transform. The two things it is doing (here) is setting the the treeherder platform (which seems reasonable to do by hand), and handle `include-nightly-version`. I'd split the later into its own transform, and the drop the `rb_taskgraph.build` transform

Reply:
Makes total sense! I started to break up `tb_taskgraph.build` in https://github.com/mozilla-mobile/reference-browser/pull/896. I'll continue there, and merge the current PR as is.